frontend/packages/ux-editor/src/components/rightMenu/RightMenu.tsx

```diff
@@ -1,29 +1,36 @@
 import React from 'react';
 import classes from './RightMenu.module.css';
-import { Tabs } from '@digdir/design-system-react';
+import { Tabs, Checkbox } from '@digdir/design-system-react';
 import { ConditionalRenderingTab } from './ConditionalRenderingTab';
 import { CalculationsTab } from './CalculationsTab';
 import { ContentTab } from './ContentTab';
 import { useTranslation } from 'react-i18next';
 import cn from 'classnames';
+import { DynamicsTab } from "./DynamicsTab";
 
 export interface RightMenuProps {
   className?: string;
 }
 
 export const RightMenu = ({ className }: RightMenuProps) => {
   const { t } = useTranslation();
+  const [showOldDynamics, setShowOldDynamics] = React.useState<boolean>(false);
```

Comment:
I think maybe we should flip this, so that the checkbox is for `showNewDynamics`. I think it will be easier to toggle it off for production that way, and we can still easily access it in dev if we want to do some testing.